controllers/operatorpolicy_status.go

```diff
@@ -394,7 +415,29 @@ func missingWantedCond(kind string) metav1.Condition {
 	}
 }
 
-// createdCond returns a Compliant condition, with a Reason like'____Created',
+// missingNotWantedCond returns a Compliant condition with a Reason like '____Excluded'
+// and a Message like 'the ____ checked by the mustnothave policy was not found'
+func missingNotWantedCond(kind string) metav1.Condition {
+	return metav1.Condition{
+		Type:    condType(kind),
+		Status:  metav1.ConditionTrue,
+		Reason:  kind + "Excluded",
+		Message: "the " + kind + " checked by the mustnothave policy was not found",
```

Comment:
I think this is easier to read.
```suggestion
		Message: "the " + kind + " checked by the must not have policy was not found",
```

Reply:
Instead of "checked" does "managed" sound better?

Reply:
Finding the right phrasing is hard. Maybe we need to keep thinking about it, I don't think these suggestions are solidly better.

I prefer keeping "mustnothave", I think it's more searchable in the docs to find exactly what this behavior is. Separate words does look better, but doesn't reflect what the actual option is in the resource.

I think "managed" could imply that the policy created/edited the resource at some point, which is not necessarily the case.

We should probably also think about whether "Excluded" is good. I just wanted a different Reason than the other condition where not found is noncompliant.